### PR TITLE
support header template.

### PR DESCRIPTION
**Pull request by hetz**
hi ichord,I want support header template.
becuse header template is a better experience
so I defined option `headerTpl`.
pls merge this pull request.

### CODE REVIEW

File: dist/css/jquery.atwho.css [hunk 1]
```diff
@@ -10,11 +10,34 @@
     border-radius: 3px;
     box-shadow: 0 0 5px rgba(0,0,0,0.1);
     min-width: 120px;
-    max-height: 200px;
-    overflow: auto;
+    /*max-height: 200px;*/
```
Comment (sp90) [inline dist/css/jquery.atwho.css]: You should never commit unused code, thats why we have github revisions

**Comment by hetz** [comment]
@sp90 that good point, I remove unused code.

**Comment by ichord** [comment]
please solve the conflicts, thanks.
BTW. I will try to switch grunt to gulp and some others building tools this weekend.

**Comment by hetz** [comment]
@ichord, get it. I solve my branch merge from upstream.

**Pull request merged**